versions.tf

```diff
@@ -15,8 +15,17 @@
  */
 
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 0.13"
```

Comment:
@morgante should we update this to 0.14 since we are now using the 0.14 image?

Reply:
No we shouldn't enforce 0.14 unless we need to.

Reply:
ack but we may not be able to detect any breakages to 0.13 automatically for future features added.